ironfish-cli/src/commands/chain/export.ts

```diff
@@ -25,14 +24,14 @@ export default class Export extends IronfishCommand {
   static args = [
     {
       name: 'start',
-      parse: (input: string): Promise<number | null> => Promise.resolve(parseNumber(input)),
+      parse: (input: string): Promise<string> => Promise.resolve(input.trim()),
       default: Number(GENESIS_BLOCK_SEQUENCE),
       required: false,
       description: 'the sequence to start at (inclusive, genesis block is 1)',
     },
     {
       name: 'stop',
-      parse: (input: string): Promise<number | null> => Promise.resolve(parseNumber(input)),
+      parse: (input: string): Promise<string> => Promise.resolve(input.trim()),
```

Comment:
This change will break the RPC below because of the typecast. We should probably fix the typecast too so this doesn't happen in the future.

Reply:
I updated the code.